### PR TITLE
Feature/sss bugfix

### PR DESCRIPTION
**Pull request by OrangeSpork**
Release Notes:

Adds SSS support for VR. SSS works with both Official and the Unofficial (KillMar) plugin.
All modes supported for both HS2 and AIS.

Notes: VR + Shadows Off + SSS in non PPO mode = Frame Rate Catastrophy. Either use PPO or turn on shadows (recommend Shadows OFF and PPO On for workable performance.
Recommended VR SSS Config: Shadows OFF, PPO On, Post Process Iterations 1, Blur Size to preference.

New Feature: Defaults

4 Defaults are included: Main Game, Maker, Studio and VR.

Maker and Studio are just the initial load in configuration. Studio scenes loaded continue to load settings and lighting from the scene as before.

As the continuous scene swapping in Main Game and VR regularly overwrites applied settings, to keep a consistent experience these presets are aggressively applied on every scene change. 

To make changes to the provided defaults, simply configure settings as desired and click the 'Save as Default' for the appropriate type button on the Preset panel. To get back to the original default, click the 'Reset Default' for the appropriate type.

Note: Provided defaults have SSS OFF (to preserve frame rates for people not using the new shader). To enable simply Load the preset, Enable SSS, Enable SSS Profile Per Object, Set Blur as desired and then Save the Default.

Note: Turn off BetterAA, having AA managed by both plugins makes things a blurry mess...

### CODE REVIEW

**Comment by ore-** [comment]
@OrangeSpork thanks for contributing to Graphics. I'm rather busy lately, will try to take a look asap. perhaps @hooh-hooah can unblock you sooner.

**Comment by hooh-hooah** [comment]
I'll merge this one if I don't have any issues while building/in the game.

File: Graphics/AIGraphics/AIGraphics.csproj [hunk 1]
```diff
@@ -32,35 +32,42 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="0Harmony, Version=2.0.0.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\packages\IllusionLibs.BepInEx.Harmony.2.0.3.1\lib\net35\0Harmony.dll</HintPath>
+      <HintPath>..\..\..\..\illusion\ais\BepInEx\core\0Harmony.dll</HintPath>
```
Comment (hooh-hooah) [inline Graphics/AIGraphics/AIGraphics.csproj]: Please don't change the reference path in the next time. We're using the NuGet Package to deal with this problem.

**Comment by hooh-hooah** [comment]
I'll merge it for now but remember, use the NuGet. 
About the default UI... I'll do something about it with the next patch.

**Pull request merged**